src/fides/api/app_setup.py

```diff
@@ -82,6 +83,9 @@ def create_fides_app(
         # Starlette bug causing this to fail mypy
         fastapi_app.add_exception_handler(FunctionalityNotConfigured, handler)  # type: ignore
     fastapi_app.add_middleware(SlowAPIMiddleware)
+    fastapi_app.add_middleware(
+        GZipMiddleware, minimum_size=1000, compresslevel=5
```

Comment:
Interesting reading about this middleware- I'm curious if there's a way to test what compress level to use in practice?

Reply:
RE: Does this mean for any API req, we can add the "Accept-Encoding": "gzip" header to get the gzipped resp? 

Yes - it is already on the call to the full experience, Jason will verify that NextJS sends it as well -